bower.json

```diff
@@ -1,18 +1,26 @@
 {
-	"name": "jquery-boilerplate",
-	"version": "4.0.0",
-	"homepage": "http://jqueryboilerplate.com",
+  "name": "jquery-boilerplate",
+  "version": "4.0.0",
+	"license": "MIT",
 	"authors": [
-		"Zeno Rocha <hi@zenorocha.com>"
-	],
-	"description": "A jump-start for jQuery plugins development.",
-	"main": "src/jquery.boilerplate.js",
+    "Zeno Rocha <hi@zenorocha.com>"
+  ],
+  "description": "A jump-start for jQuery plugins development.",
+	"homepage": "http://jqueryboilerplate.com",
 	"keywords": [
-		"jquery",
-		"plugin",
-		"boilerplate",
-		"jquery-plugin",
-		"jquery-boilerplate"
-	],
-	"license": "MIT"
+    "jquery",
+    "plugin",
+    "boilerplate",
+    "jquery-plugin",
+    "jquery-boilerplate"
+  ],
+  "main": "src/jquery.boilerplate.js",
+	"dependencies": {
+    "jquery": "1"
+  },
+  "ignore": [
+    "npm-debug.log",
+		"bower_components",
```

Comment:
removing the "license" from package.json and bower.json will throw error when the package will be installed via npm or Bower. Keep it with the copy of the license you must provide!